config.js

```diff
@@ -27,6 +27,31 @@ const templateData =
         `<li><a href = '${item.pathname}'>${item.title}</a></li>`);
 
       return `<ul>${items.join("")}</ul>`;
+    },
+    getSubPages: function (nav)
```

Comment:
Suggestion: We can also introduce second parameter `level`. To help with sidebar implementation, something like:
```
getSubPages: function (nav, level = 100) {
  ...
  if (subPages.length && --level)
  ...
}
```

This way we should be able to get only children of the existing page, which can be useful in sidebar, ex.:
```
site.getSubPages(otherPage, 1);
```

Reply:
Or maybe:
```
getSubPages: function (nav, grandChildren) {
...
  if (subPages.length && grandChildrenl)
...
```